app/assets/scss/patterns/_pricing.scss

```diff
@@ -0,0 +1,81 @@
+.pricing {
+
+  &-column {
```

Comment:
Can we lose this `&-` format? It makes searching for class names you find in the generated CSS difficult.